content/articles/2018-07-talk-carpe-diem.markdown

```diff
@@ -0,0 +1,82 @@
+
+Some creators take a peculiar pride in being "bad at marketing", but it is not some elusive skill. Nor is marketing ability anti-correlated with creating a high-quality product. You can acknowledge the value in marketing or you can declare that cultivating broader usage is not your goal. Pick one.
+
+## Where does your thing live?
+
+Determine the primary online home for your product. Possibilities:
+
+  * A custom website, possibly made with [pkgdown](http://pkgdown.r-lib.org), [blogdown](https://bookdown.org/yihui/blogdown/), [bookdown](https://bookdown.org/yihui/bookdown/), or, in the case of [Shiny](https://shiny.rstudio.com), a [running instance of the app itself](https://www.shinyapps.io)
+  * A development venue, such as a [GitHub repository](https://github.com)
+  * A [CRAN](https://cran.r-project.org/web/packages/index.html) package landing page
+
+Visit this home with the eyes of a newcomer and refine accordingly. If your thing has multiple online homes, make sure they link to each other, with a strong current towards the primary home.
+
+## Make a shortlink
+
+Unless your primary URL is already extremely short and memorable, consider creating a shortlink that is both. A link in a talk will be captured in cell phone photos and typed with fingers into phones. Make it easy on people!
+
+http://bit.ly
+
+## Title and closing slide
+
+Make a slide with the most important details on you and your thing. Include this very early in your talk (probably first?) and repeat some variant at the end. This slide should feature your shortlink and any relevant URLs or handles, e.g. Twitter, GitHub, website, or email address.
+
+DO THIS IN AN ABSOLUTELY HUGE FONT. Now is not the time for subtlety or mumbling "I know it's hard to read, but ...".
+
+Leave this slide up for a bigly amount of time, so that people have ample opportunity to get out their phones and fire up the camera. Do not flash this vital slide for 3 seconds, only to spend 2 minutes displaying a bulleted outline of your talk.
+
+## Feature slides
+
+Admit that people photograph and share individual slides, e.g., via Twitter. This can be a good thing, especially if you harness it for your goals! If an audience member is moved to share your thing, it is natural they want to enhance the basic message with an evocative slide or image. Think about how well your slides work with this well-established behaviour.
+
+If something would be terrible taken out of context or circulated widely, reflect on that and take appropriate measures.
+
+If you have a pithy and hilarious point, give that slide some extra attention.
+
+*Link to some guides on live-tweeting? If you'd be chuffed to see people tweet about your talk, view your talk through their eyes. This is like grant-writing: notice the words the funder uses to describe what they want to fund and repeat them back, verbatim. Find out what a live-tweeter needs and give them exactly that.*
```

Comment:
Emily Robinson has tips on live tweeting in this post: http://hookedondata.org/Building-Your-Data-Science-Network-Finding-Community/ (possible that Dave Robinson actually has slides on live tweeting, I'll ask)

Reply:
Or just the tweet of drob's slide on live-tweeting, I guess: https://twitter.com/zabormetrics/status/855518845779496961?ref_src=twsrc%5Etfw%7Ctwcamp%5Etweetembed%7Ctwterm%5E855518845779496961&ref_url=http%3A%2F%2Fvarianceexplained.org%2Fr%2Fnyr-conference%2F